tmt/convert.py

```diff
@@ -156,11 +157,11 @@ def read_nitrate(beaker_task, common_data):
         data = dict()
         echo("test case found '{0}'.".format(testcase.identifier))
         # Test identifier
-        data['tcms'] = testcase.identifier
+        data['extra-tcms'] = testcase.identifier
         # Beaker task name (taken from summary)
         if testcase.summary:
-            data['task'] = testcase.summary
-            echo(style('task: ', fg='green') + data['task'])
+            data['extra-summary'] = testcase.summary
```

Comment:
This rename would mean there would be 2 summaries `data['summary']` and `data['extra-summary']` which would be even more confusing than it is already is (beacause of the mapping of nitrate to tmt)

Reply:
Well, I think this `extra-summary` makes more sense because it is the old tcms test case summary. So if you are looking for the old summary you should find it better now. Or do you have any better name for that? Or shall we just forget about the old summary? Or override `summary` from Makefile?

Reply:
Thinking about it a bit more I would vote for removing `extra-summary` altogether. If you need to distinguish two test cases you can (and should) include this information in the `summary`. That is the primary field for quick overview about the test. I think it does not make sense to create this duplication.

Reply:
I get that I just don't like that there are two summaries, however I cannot think of better name so I suppose it will have to be this way.

Reply:
Ok, then let's keep it for now. @lukaszachy shared some more use cases which need to be covered by this field. So we'll have to live with this for some time as a temporary solution.